src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -240,10 +240,16 @@ public ImageProcessor(ImageClassificationTransformer transformer)
 
             public Tensor ProcessImage(in VBuffer<byte> imageBuffer)
             {
-                var imageTensor = EncodeByteAsString(imageBuffer);
-                var processedTensor = _imagePreprocessingRunner.AddInput(imageTensor, 0).Run()[0];
-                imageTensor.Dispose();
-                return processedTensor;
+                using (var imageTensor = EncodeByteAsString(imageBuffer))
+                    try
+                    {
+                        var processedTensor = _imagePreprocessingRunner.AddInput(imageTensor, 0).Run()[0];
+                        return processedTensor;
+                    }
+                    catch (TensorflowException)
```

Comment:
>TensorflowException [](start = 27, length = 19)

TensorflowException covers broad cases of exception, the only exception we want to swallowis the on for bad images, please ONLY catch those exception and rethrow the rest.